AlfrescoApp/View Controllers/FileFolder View Controller/BaseCollectionViewFlowLayout.m

```diff
@@ -141,8 +142,15 @@ - (NSArray *)layoutAttributesForElementsInRect:(CGRect)rect
             attributes.editing = self.isEditing;
             attributes.isSelectedInEditMode = [self.dataSourceInfoDelegate isItemSelected:attributes.indexPath];
             attributes.thumbnailWidth = self.thumbnailWidth;
-            attributes.shouldShowSeparatorView = (self.numberOfColumns == 1) ? YES : NO;
-            attributes.shouldShowAccessoryView = (self.numberOfColumns == 1) ? YES : NO;
+            attributes.shouldShowSeparatorView = (self.numberOfColumns == 1);
+            attributes.shouldShowAccessoryView = (self.numberOfColumns == 1);
+            attributes.shouldShowNodeDetails = (self.numberOfColumns == 1);
+            attributes.shouldShowEditBelowContent = (self.numberOfColumns == 1);
+            attributes.shouldShowSmallThumbnailImage = (self.numberOfColumns == 1);
+            attributes.nodeNameHorizontalDisplacement = (self.numberOfColumns == 1)? 58 : 10;
+            attributes.nodeNameVerticalDisplacement = (self.numberOfColumns == 1)? 10 : self.thumbnailWidth + 20;
+            attributes.nodeNameFont = (self.numberOfColumns == 1)? [UIFont systemFontOfSize:17] : [UIFont fontWithName:@"HelveticaNeue-Light" size:13];
+            attributes.editImageTopSpace = (self.numberOfColumns == 1) ? 17 : 0;
```

Comment:
As above, these hardcoded numbers need explaining. I'm picking these up because often there's an implied relationship between them, or perhaps an assumption of one, which needs to be made clear in the source code. For example: does the value "17" here need to match the value "17" in the line above?